### ISSUE TITLE
Accessibility violation for ibm-label

### ISSUE THREAD

**Issue by maheshkanchetti** [IBM/carbon-components-angular#1501]
ibm-label component
The following accessibility violation is fixed
The <label> element does not have descriptive text that identifies the expected input

https://github.com/IBM/carbon-components-angular/pull/1502

**Comment by Donisius** [comment]
Closed by #1502
